SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslationManager/SpatialVirtualWorkerTranslationManagerTest.cpp

```diff
@@ -1,152 +0,0 @@
-// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Do we not test the virtual worker translation manager now? Is there a ticket to rewrite these?

Reply:
These tests were (subjectively) bad. Essentially they validated that taking an authority op resulted in making a entity query. The way the internal VWTM works now means this isn't the case (authority ops result in partition entities being created, and then an entity query), and there's currently no way to automate entity creation.